caffe2/CMakeLists.txt

```diff
@@ -344,53 +344,6 @@ endif()
 
 
 if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
-  if(USE_DISTRIBUTED)
-
-    # Define this target even if we're building without TensorPipe, to make life
-    # easier to other targets that depend on this. However, in that case, by not
-    # setting the USE_TENSORPIPE compile definition, this target will just end
-    # up being empty. Downstream targets should also add a #ifdef guard.
-    if(NOT WIN32)
-      add_library(process_group_agent
```

Comment:
Here we were defining some ad-hoc targets that contained the PG and TP agents without pulling in the rest of libtorch_python, for use in our C++ tests. Now we can just drop these and instead use libtorch directly.